src/components/FormAdd.jsx

```diff
@@ -1,22 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FormAdd.css';
 import { useDispatch } from 'react-redux';
 import { addBook } from '../redux/books/BookSlice';
 
+let counter = 2;
 function Form() {
-  // const books = useSelector((state) => state.books.books);
+  function generateId() {
+    counter += 1;
+    return counter;
+  }
+
   const dispatch = useDispatch();
 
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handlechangTitle = (event) => {
+    setTitle(event.target.value);
+  };
+  const handleAuthor = () => {
+    setAuthor('John Smith');
+  };
+
   const handleAddBook = () => {
     if (document.querySelector('.inpText').value !== '') {
+      handleAuthor();
       const selectElement = document.getElementById('dd');
       const selectedOption = selectElement.options[selectElement.selectedIndex];
```

Comment:
- [ ] It would be better if we address the previous comment, here we are building the app with `React` so we need to follow the React way in building the app. Kindly desist from directly manipulating the DOM as mentioned earlier